src/Components/stateCard/StateCard.jsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Flex, Text } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+
```

Comment:
- [ ] It would be better if we write unit test for our redux pure functions, also lets us add negative test cases so we ensure our application works as expected

Reply:
okay thanks for your great effort

Reply:
- [ ] Tests written